src/itensor.jl

```diff
@@ -419,8 +419,8 @@ avoiding allocating new data if possible.
 For example, an ITensor with Diag storage will become Dense storage,
 filled with zeros except for the diagonal values.
 """
-function NDTensors.dense(A::ITensor)
-  T = dense(tensor(A))
+function dense(A::ITensor)
```

Comment:
What was the reason for this change? I think we want the original version. The new version would mean that both ITensors and NDTensors would have their own version of `dense`, instead of a single version in NDTensors that is overloaded by ITensors.

Reply:
Thanks for noticing - the reason for this change was that I was getting a lot of strange errors about `dense` not being defined. A lot of things I tried didn't work. Let's discuss on Slack because I'm probably thinking about it the wrong way.

Reply:
Ah, I think you just want to make sure that you overload this version and the `AbstractMPS` version with `NDTensors.dense`.

I believe the problem was that since you defined the `AbstractMPS` version as `dense(M::AbstractMPS)` instead of `NDTensors.dense(M::AbstractMPS)`, it creates a new local version of the `dense` function within the ITensors module, so then Julia can't "see" the NDTensors version of the `dense` function without calling it as `NDTensors.dense`.